pkg/tfbridge/info.go

```diff
@@ -172,6 +175,7 @@ type DataSourceInfo struct {
 func (info *DataSourceInfo) GetTok() tokens.Token              { return tokens.Token(info.Tok) }
 func (info *DataSourceInfo) GetFields() map[string]*SchemaInfo { return info.Fields }
 func (info *DataSourceInfo) GetDocs() *DocInfo                 { return info.Docs }
+func (info *DataSourceInfo) GetImportDetails() string          { return "" }
```

Comment:
I don't like we are adding a member to an interface which is not applicable to one of its implementations (because data sources do not have Import sections), but I understand we also have to be pragmatic and ship.

Could leave a comment indicating the design decision here.

Reply:
Agreed - I don't love this either